src/libsync/propagateremotemove.cpp

```diff
@@ -254,6 +255,14 @@ void PropagateRemoteMove::finalize()
             newItem._size = oldRecord._fileSize;
         }
     }
+
+    const auto targetFile = propagator()->fullLocalPath(_item->_renameTarget);
```

Comment:
@mgallien Could you explain what happens here and why?

Reply:
usually we only rename files on the server that also exists on local sync folder
in my case I would like to do the rename before downloading the file
this is why the local file may not exist even if we rename it on server